### ISSUE TITLE
Errors upgrading to 0.8 due to wrong command_attr version in Cargo.lock

### ISSUE THREAD

**Issue by krial057** [serenity-rs/serenity#810]
**TL;DR**
This problem is similar to #766 . However I had the opposite problem. I updated my serenity dependency from 0.7.4 to 0.8 and switched to the group macro. Then I got the following errors:

> error: cannot find attribute `group` in this scope 
> error: cannot find attribute `commands` in this scope

**Explanation**
As Cargo.toml from serenity depends on 0.1 of command_attr, the command_attr in the Cargo.lock was still set to 0.1.6 after the update to serenity 0.8. command_attr 0.1.6 doesn't contain the group attribute yet and therefore the compiler errors.

**Impact**
Users updating the dependency without running `cargo update` afterwards will encounter the same problems.

**Suggestion**
One of the following:

- Fix the dependency in the serenity Cargo.toml to command_attr=0.1.7
- Update command_attr to 0.2 and update Cargo.toml of serenity to command_attr=0.2
- Write in the README somewhere to not forget executing `cargo update` after updating the dependency. Leave this issue open so that users with the same problem have it easier to fix the problem.

**Comment by Lakelezz** [comment]
This should be fixed. Once you use Serenity 0.8, Cargo should pick command_attr 0.2.0.
Also, @acdenisSK yanked the faulty version.

If there are issues, let us know!
